py-polars/tests/unit/series/test_series.py

```diff
@@ -1363,6 +1363,11 @@ def test_filter() -> None:
     assert_series_equal(s.filter([True, False, True]), pl.Series("a", [1, 3]))
 
 
+def test_filter_element() -> None:
+    s = pl.Series("a", [1, 2, 3])
+    assert_series_equal(s.filter(pl.element() < 3), pl.Series("a", [1, 2]))
+
+
```

Comment:
How about just add the test line to the `test_filter()` function, and use the same `pl.element() !=2` as in the function docstring to make it more consistent? i.e.
```python
assert_series_equal(s.filter(pl.element() != 2), pl.Series("a", [1, 3]))
```

Reply:
Consinstency with the doctest is not very important, but I'll beef up the test a bit.